src/popup/router/components/TransferSendForm.vue

```diff
@@ -39,7 +40,12 @@
         </a>
       </template>
     </InputField>
-
+    <div class="status">
```

Comment:
This could have been achieved but we have now deleted `error` and `warning` props. I have mentioned the new approach in the other conversation in here https://github.com/aeternity/superhero-wallet/pull/1548#discussion_r963030701